spotify_dl/spotify_dl.py

```diff
@@ -50,14 +50,19 @@ def spotify_dl():
             config = json.loads(file.read())
 
         for key, value in config.items():
-            if value and (value.lower() in ['true', 't']):
+            if (isinstance(value, bool) and value) or (isinstance(value, str) and value and value.lower() in ['true', 't']):
                 setattr(args, key, True)
             else:
                 setattr(args, key, value)
 
     if args.verbose:
         log.setLevel(DEBUG)
 
+    if not hasattr(args, 'url'):
+        raise(Exception("No playlist url provided"))
+    if not hasattr(args, 'o'):
```

Comment:
this should be `output`. Caught this after merging. opened #235

Reply:
Ah crap. missed that.